### PR TITLE
Add EFFECT_TRACK

### PR DESCRIPTION
**Pull request by fdarosa2663**

### CODE REVIEW

**Comment by fdarosa2663** [comment]
Adding 3 more effects that will be used in my prop to allow user customization of features

File: common/saber_base.h [hunk 1]
```diff
@@ -38,10 +38,19 @@ extern SaberBase* saberbases;
     DEFINE_EFFECT(POWERSAVE)                    \
     DEFINE_EFFECT(BATTERY_LEVEL)                \
     DEFINE_EFFECT(FAST_ON)                      \
+    DEFINE_EFFECT(QUOTE)			\
+    /* Fett263 Prop Effects */                  \
```
Comment (profezzorn) [inline common/saber_base.h:42]: No, effects are not just for one prop.
Please remove this comment.

**Comment by profezzorn** [comment]
The title of this PR is not very accurate now since you added a bunch more effects to it...

File: common/saber_base.h (hunk 1 above)
Comment (fdarosa2663) [reply common/saber_base.h]: ok, removed thought you'd ask me to add since they're all planned for my prop

**Comment by fdarosa2663** [comment]
Sorry, yeah expanded- if you want a separate PR for these let me know, figured they were related.

**Pull request merged**